+++ minhook/meson.build
@@ -0,0 +1,18 @@
+project('minhook', 'c', version: '1.0.0')
+
+minhook_inc = include_directories('include')
+minhook_lib = static_library(
+	'minhook',
+	include_directories: minhook_inc,
+	sources: [
+		'src/buffer.c',
+		'src/hook.c',
+		'src/trampoline.c',
+		'src/hde/hde32.c',
+		'src/hde/hde64.c'
+	]
+)
+minhook_dep = declare_dependency(
+	link_with: minhook_lib,
+	include_directories: minhook_inc,
+)
